upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -833,11 +833,34 @@ has_craycm() {
   helm list -n "${ns}" --filter 'cray-certmanager$' | grep cray-certmanager > /dev/null 2>&1
 }
 
-state_name="UPGRADE_CERTMANAGER_CHART"
+state_name="UPGRADE_CERTMANAGER_0141_CHART"
 state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
 if [[ ${state_recorded} == "0" && $(hostname) == "${PRIMARY_NODE}" ]]; then
   echo "====> ${state_name} ..." | tee -a "${LOG_FILE}"
   {
+    # The actions below only need to happen iff we are on 0.14.1 if we're
+    # already at 1.5.5+ the crd ownership changes makes this logic impossible to
+    # work due to helm hooks. Making this work on both isn't really worth the
+    # time so just constrain this block of logic to 0.14.1 where we know its
+    # needed.
+    gate="0.14.1"
+    found=$(helm list -n cert-manager --filter 'cray-certmanager$' | awk '/deployed/ {print $10}')
+
+    if [ "${found}" = "${gate}" ]; then
+      printf "note: found old cert-manager that needs upgrades %s\n" "${found}" >&2
+    else
+      printf "note: cert-manager helm chart version %s\n" "${found}" >&2
+
+      # We might be rerunning from a pre 1.5.x install and there is no
+      # cert-manager installed due to a prior removal
+      if [ "${found}" = "" ]; then
+	printf "note: no helm install appears to exist for cert-manager, likely this state is being run again\n" >&2
+      else
+        printf "note: no cert-manager upgrade steps needed 0.14.1 cert-manager not installed\n" >&2
```

Comment:
```suggestion
        printf "note: no cert-manager upgrade steps needed as 0.14.1 cert-manager was not installed\n" >&2
```

Reply:
not against it but was probably would be better phrased as is no? More the version installed versus what was installed before, in this case its just comparing if we are or aren't 0.14.1 to gate the run.

Reply:
The original statement was a bit confusing to me, whatever change you think will make it clearer is fine with me.